packages/xo-web/src/xo-app/settings/remotes/remote.js

```diff
@@ -338,6 +355,17 @@ export default decorate([
                   value={host}
                 />
               </div>
+              <div className='input-group '>
+                <input
+                  className='form-control'
+                  name='protocol'
+                  onChange={effects.linkState}
+                  pattern='(?!^(\d{1,3}\.){3}\d{1,3}$)(^[a-z0-9]([a-z0-9-]*(\.[a-z0-9])?)*$)'
```

Comment:
Is it the same pattern as the `bucket` input on purpose or is it supposed to be something else?